recod_erpnext_design/fixtures/terms_and_conditions.json

```diff
@@ -0,0 +1,50 @@
+    "title": "Payment on delivery by bank transfer"
+  },
+  {
+    "disabled": 0,
+    "docstatus": 0,
+    "doctype": "Terms and Conditions",
+    "modified": "2020-06-15 09:29:57.436782",
+    "name": "Payment on receipt by bank transfer",
+    "parent": null,
+    "parentfield": null,
+    "parenttype": null,
+    "terms": "<h4>{{_(\"Terms and Conditions\")}}</h4><div class=\"row data-field\" data-fieldname=\"payment_conditions\" data-fieldtype=\"Data\"><div class=\"col-xs-5\"><label>{{_(\"Payment conditions\")}}</label></div><div class=\"col-xs-7 text-left value\">{{_(\"on receipt\")}}</div></div>{% set links=frappe.get_all('Bank Account', filters={'company': company, 'is_default': 1}, fields=['company', 'bank', 'iban', 'bank_account_no', 'branch_code', 'swift_number'] ) %}{% if links %}{% set bank_account=links[0] %}{% if bank_account %}<strong>{{_(\"Payment by transfer to the following bank account:\")}}</strong>{% if bank_account.company %}<div class=\"row data-field\" data-fieldname=\"company\" data-fieldtype=\"Link\"><div class=\"col-xs-5\"><label>{{_(\"Bank Account\")}}</label></div><div class=\"col-xs-7 text-left value\">{{bank_account.company}}</div></div>{% endif %}{% if bank_account.bank %}<div class=\"row data-field\" data-fieldname=\"bank\" data-fieldtype=\"Link\"><div class=\"col-xs-5\"><label>{{_(\"Bank\")}}</label></div><div class=\"col-xs-7 text-left value\">{{bank_account.bank}}</div></div>{% endif %}{% if bank_account.iban %}<div class=\"row data-field\" data-fieldname=\"iban\" data-fieldtype=\"Data\"><div class=\"col-xs-5\"><label>{{_(\"IBAN\")}}</label></div><div class=\"col-xs-7 text-left value\">{{bank_account.iban}}</div></div>{% endif %}{% if bank_account.bank_account_no %}<div class=\"row data-field\" data-fieldname=\"bank_account_no\" data-fieldtype=\"Data\"><div class=\"col-xs-5\"><label>{{_(\"Bank Account No\")}}</label></div><div class=\"col-xs-7 text-left value\">{{bank_account.bank_account_no}}</div></div>{% endif %}{% if bank_account.branch_code %}<div class=\"row data-field\" data-fieldname=\"branch_code\" data-fieldtype=\"Data\"><div class=\"col-xs-5\"><label>{{_(\"Branch Code\")}}</label></div><div class=\"col-xs-7 text-left value\">{{bank_account.branch_code}}</div></div>{% endif %}{% if bank_account.swift_number %}<div class=\"row data-field\" data-fieldname=\"swift_number\" data-fieldtype=\"Data\"><div class=\"col-xs-5\"><label>{{_(\"SWIFT number\")}}</label></div><div class=\"col-xs-7 text-left value\">{{bank_account.swift_number}}</div></div>{% endif %}{% endif %}{% endif %}",
+    "title": "Payment on receipt by bank transfer"
+  }
+]
```

Comment:
Where are the [Termsfeed](https://app.termsfeed.com/) _Terms and Conditions_ ??

Reply:
@AminovE99 I'm going merge this anyway, but you missed this and it needs to be added later.